### DISCUSSION TITLE
Ultima: compact not working

### DISCUSSION THREAD

**Discussion by konagithub2** [primefaces/community#1326] [PrimeFaces Templates]
Hi,

can you please help me with the compact stylesheet. 
ultima 6.1 PF 12.0.7
I am using ultima-blue in web.xml
I updated the body with layout-compact.
But still nothing. When I try to compile just the _compact.scss file then I am getting plenty of errors:

Error: Interpolation isn't allowed in plain CSS.
    ╷
148 │   background: url("#{resource['demo:images/pages/accessDenied-bg.jpg']}");
    │                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ╵
  src\main\webapp\resources\demo\styles\demo.css 148:20  root stylesheet

Do I need to compile this file and where are the variables for the compact layout?

Thanks for your help.

**Comment by melloware** [comment]
What exactly are you trying to do?  I have the Ultima template working fine?

**Comment by melloware** [comment]
Closing as no response from OP
